jmclient/jmclient/configure.py

```diff
@@ -508,16 +551,30 @@ def get_irc_mchannels():
         return _get_irc_mchannels_old()
 
     SECTION_NAME += ':'
-    irc_sections = []
+    sections = []
     for s in jm_single().config.sections():
         if s.startswith(SECTION_NAME):
-            irc_sections.append(s)
-    assert irc_sections
+            sections.append(s)
+    assert sections
 
-    req_fields = [("host", str), ("port", int), ("channel", str), ("usessl", str)]
+    irc_fields = [("host", str), ("port", int), ("channel", str), ("usessl", str),
+              ("socks5", str), ("socks5_host", str), ("socks5_port", str)]
```

Comment:
Yep. It's cast to `int` in the `IRCMessageChannel` constructor after being passed over AMP ... so this could easily have been a copy paste error from ages ago that didn't matter because of that cast (reading from the config objects, you always get str unless you do `config.getint(..)`). I'll fix it and it shouldn't break.

Reply:
Done in 4cff6d7bb1e3f81473c99dccd3460aae2f2a6c55